src/main/java/frc/trigon/robot/subsystems/elevator/placeholderelevator/PLACEHOLDERElevatorIO.java

```diff
@@ -0,0 +1,68 @@
+package frc.trigon.robot.subsystems.elevator.placeholderelevator;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Voltage;
+import frc.trigon.robot.subsystems.elevator.ElevatorConstants;
+import frc.trigon.robot.subsystems.elevator.ElevatorIO;
+import frc.trigon.robot.subsystems.elevator.ElevatorInputsAutoLogged;
+import frc.trigon.robot.utilities.Conversions;
+
+public class PLACEHOLDERElevatorIO extends ElevatorIO {
+    private final TalonFX
+            masterMotor = PLACEHOLDERElevatorConstants.MASTER_MOTOR,
+            followerMotor = PLACEHOLDERElevatorConstants.FOLLOWER_MOTOR;
+    private final MotionMagicVoltage positionRequest = new MotionMagicVoltage(0).withEnableFOC(PLACEHOLDERElevatorConstants.FOC_ENABLED);
+    private final VoltageOut voltageRequest = new VoltageOut(0).withEnableFOC(PLACEHOLDERElevatorConstants.FOC_ENABLED);
+
+    @Override
+    protected void updateInputs(ElevatorInputsAutoLogged inputs) {
+        refreshStatusSignals();
+        inputs.motorVoltage = PLACEHOLDERElevatorConstants.MOTOR_VOLTAGE_STATUS_SIGNAL.getValue();
+        inputs.positionMeters = getEncoderPositionMeters();
+        inputs.velocityMetersPerSecond = getEncoderVelocityMetersPerSecond();
+        inputs.profiledSetpointMeters = positionRequest.Position;
+    }
```

Comment:
this is not the profiled position (you need to closed loop reference), and this is not in meters.